tests/kubectl/kubectl.go

```diff
@@ -2,16 +2,24 @@ package kubectl
 
 import (
 	"bytes"
+	"os"
 	"os/exec"
 )
 
 func Kubectl(command ...string) (string, string, error) {
 	var stdout, stderr bytes.Buffer
 	cmd := exec.Command("./cluster/kubectl.sh", command...)
-	cmd.Dir = ".."
+	cmd.Dir = getClusterRootDirectory()
```

Comment:
Let's parameterize the entire command (similar to [here](https://github.com/nmstate/kubernetes-nmstate/blob/794179d34d0b09697707a5927789b80f1c218129/test/cmd/kubectl.go#L7))
suggestion:
```
kubectl := environment.GetVarWithDefault("KUBECTL", "../cluster/kubectl.sh")
cmd := exec.Command(kubectl, command...)
...
```

then export the KUBECTL entire command like in knmstate automation [here](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/automation/check-patch.e2e-nmstate-functests.sh#L40)

Reply:
thanks but what vars you set on KMP itself (when you run it on kmp itself and not with CNAO cluster) ?
i think it won't work on kmp itself out of the box, because the cmd.Dir is missing, unless you add another var that solves it